avocado/core/status/server.py

```diff
@@ -38,7 +39,10 @@ async def create_server(self):
     async def serve_forever(self):
         if self._server_task is None:
             await self.create_server()
-        await self._server_task.serve_forever()
+
+        # TODO: Delete this after deprecation of python 3.6
+        if sys.version_info >= (3, 7):
+            await self._server_task.serve_forever()
```

Comment:
This is strange... because we have python 3.6 on travis making our tests and also python3.6 docs shows a `serve_forever()` method:

https://docs.python.org/3.6/library/socketserver.html#socketserver.BaseServer.serve_forever

Reply:
Unfortunately I didn't find in the documentation that the [asyncio.Server](https://docs.python.org/3.6/library/asyncio-eventloop.html#asyncio.Server) inherits from [socketserver.BaseServer](https://docs.python.org/3.6/library/socketserver.html#socketserver.BaseServer.serve_forever) but maybe I am missing something. I didn't find why Travis didn't catch that but maybe we have some problem in our testing.

Reply:
We can make a draft from this and investigate the problem further.